src/aof.c

```
@@ -2384,6 +2387,7 @@ int rewriteAppendOnlyFile(char *filename) {
     /* Make sure data will not remain on the OS's output buffers */
     if (fflush(fp)) goto werr;
     if (fsync(fileno(fp))) goto werr;
+    if (reclaimFilePageCache(fileno(fp), 0, 0) == -1) goto werr;
```

if we wanna silently ignore the error, we should drop that `goto`